src/pages/login.tsx

```diff
@@ -4,25 +4,27 @@ import { DISCORD, ENV } from '../constants/constants';
 import { axios } from '../utils/HttpService';
 
 const Login: React.FC = () => {
-  const search = window.location.search;
-  const params = new URLSearchParams(search);
-  const code = params.get('code');
-  const baseUrl = window.location.origin;
+  React.useEffect(() => {
+    // Had to put this the useEffect in cause window is only on client side and this was rendered server side unless in useEffect
+    const search = window.location.search;
+    const params = new URLSearchParams(search);
+    const code = params.get('code');
+    const baseUrl = window.location.origin;
 
-  const login = async () => {
-    if (!code) return;
-    try {
-      const res: any = await axios.post(`${ENV.API_URL}/login`, {
-        code,
-        redirectUrl: `${baseUrl}/login`,
-      });
-    } catch (err: any) {
-    } finally {
-      window.location.replace(baseUrl);
-    }
-  };
+    const login = async () => {
+      if (!code) return;
+      try {
+        const res: any = await axios.post(`${ENV.API_URL}/login`, {
+          code,
+          redirectUrl: `${baseUrl}/login`,
+        });
+      } catch (err: any) {
+        console.log(err);
```

Comment:
Guessing this is a debugging leftover?

Reply:
uhhh yarn build was annoyed that the catch was empty so i just logged it LOL

Reply:
Should be able to just have a try finally right?